src/bicep/modules/privateLink.bicep

```
@@ -0,0 +1,211 @@
+@description('The name of the resource the private endpoint is being created for')
+param laWorkspaceName string
+
+@description('The resource id of the resoure the private endpoint is being created for')
+param laWorkspaceResourceId string
+
+@description('The region the private endpoint will be created in')
+param privateEndpointLocation string = resourceGroup().location
```

I think we try to keep modules as unopinionated of implementation details as possible
```suggestion
param location string
```